ocean_lib/web3_internal/constants.py

```diff
@@ -19,3 +19,13 @@
     42: "Kovan",
     100: "xDai",
 }
+
+NETWORK_TIMEOUT_MAP = {
```

Comment:
@DMats when reviewing, what do you think?
Should we combine the timeout and name maps? I think this is more readable, but on the other hand it would also work if we have e.g.

{1: {"name": "Mainnet", "timeout": 10*60}}

Also, regarding specific timeouts for each network, CC @trentmc, what are some values I should use?

Reply:
Use the same timeout values as ocean.js.

I believe Alex C said it was ten min for eth mainnet.

For others, they would be probably 1 s for ganache, and about 30 s for the rest. But check ocean.js first.

Reply:
@calina-c I think we should keep them separate (the way it is), for readability. If a 3rd piece of data is added in the future, then we should combine them into a single data structure at that point.

Reply:
I was just looking at both network maps. Right now it's out of line with what _we_ actually support.

- morden, kovan and xdai are listed, yet we don't support them
- polygon mainnet is not listed, yet we need to support it

I recommend only putting in what we actually support. No more and no less.

Reply:
polygon is added to the `SUPPORTED_NETWORK_NAMES` as part of #308

Reply:
Alex said the timeouts are ok. However, let's wait for isConsumable (easier to test in provider) and #308, to unify everything together. Marking this as blocked for now (no pun intended 😁 ).